ambry-api/src/main/java/com.github.ambry/clustermap/DiskId.java

```diff
@@ -24,19 +24,19 @@
    *
    * @return absolute mount path.
    */
-  public String getMountPath();
+  String getMountPath();
 
   /**
    * Gets the state of the DiskId.
    *
    * @return state of the DiskId.
    */
-  public HardwareState getState();
+  HardwareState getState();
 
   /**
    * Gets the raw capacity in bytes for this DiskId.
    *
    * @return the raw capacity in bytes
    */
-  public long getRawCapacityInBytes();
```

Comment:
Not sure if you want to make the changes to remove `public` piecemeal. Also seems like it has been done in some places but not elsewhere (for e.g. `ReplicaId`).